src/pdmat.jl

```diff
@@ -68,8 +68,15 @@ end
 
 ### quadratic forms
 
-quad(a::PDMat, x::AbstractVector) = dot(x, a * x)
-invquad(a::PDMat, x::AbstractVector) = dot(x, a \ x)
+function quad(a::PDMat, x::AbstractVector)
+    z = a.chol.U * x
+    dot(z, z)
```

Comment:
You might even be able to speed this up a bit using `sum(abs2, z)`. I've found that is often faster than `dot(z, z)`.